config/esbuild/build.js

```diff
@@ -1,51 +1,51 @@
 #!/usr/bin/env node
-const pathsResolverPlugin = require("./paths_resolver_plugin")
-const globImportPlugin = require("./glob_import_plugin")
-const postcss = require("postcss")
-const sass = require("sass")
-const url = require("postcss-url")
-const fs = require("fs")
+const pathsResolverPlugin = require('./paths_resolver_plugin');
```

Comment:
Hi sam! It looks like you are not anymore using our predefined settings for formatting the js files. Please use our predefined settings.

Reply:
Reverted the file. Please have a look. https://github.com/sirajudheenam/elektra/blob/ec2_credential_fix_26_oct_23/config/esbuild/build.js